controllers/profiles/dev/profile_dev_test.go

```diff
@@ -391,6 +404,7 @@ func Test_VolumeWithCapitalizedPaths(t *testing.T) {
 	workflow := test.GetSonataFlow(test.SonataFlowGreetingsWithStaticResourcesCR, t.Name())
 
 	client := test.NewSonataFlowClientBuilder().WithRuntimeObjects(workflow, configMap).WithStatusSubresource(workflow, configMap).Build()
+	knative.SetDisvoveryClient(test.CreateFakeKnativeDiscoveryClient())
```

Comment:
This naming will be replaced once you accept my changes. `SetDiscoveryClient`.

Reply:
Done